README.md

```diff
@@ -587,6 +588,10 @@ Comparison of NoSQL servers: http://kkovacs.eu/cassandra-vs-mongodb-vs-couchdb-v
 * [Tiger VNC](http://tigervnc.org/) - TigerVNC is a high-performance, platform-neutral implementation of VNC (Virtual Network Computing), a client/server application that allows users to launch and interact with graphical applications on remote machines.
 * [X2go](http://wiki.x2go.org/doku.php) - X2Go is an open source remote desktop software for Linux that uses the NX technology protocol.
 
+### Router
+
+* [OPNsense](https://opnsense.org/) - An open source FreeBSD-based firewall and router with traffic shaping, load balancing, and virtual private network capabilities.
```

Comment:
-  Format your submission as follows, where Demo and Clients are optional:
``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

(missing source code link, license and language. I suggest https://github.com/opnsense as the source code link, `BSD-2-Clause` as the license, and `C/PHP` as the language tag)

- Any category you are creating has the minimum requirement of 3 items.

Since there is no other appropriate category yet, I suggest adding [PfSense CE](https://en.wikipedia.org/wiki/PfSense) and [DD-WTR](https://en.wikipedia.org/wiki/DD-WRT) and/ord [OpenWrt](https://en.wikipedia.org/wiki/OpenWrt) alongside it to comply with this guideline

Reply:
In that context I would suggest to add https://openwisp.org/ as well.. but not sure where :-) ... OpenWISP is for centrally managing/orchestrating the OpenWrt instances (100s or more..)

Reply:
> In that context I would suggest to add https://openwisp.org/ as well.. but not sure where :-) ... OpenWISP is for centrally managing/orchestrating the OpenWrt instances (100s or more..)

Maybe in Network Configuration Management?

Reply:
> OpenWISP

Please open a separate issue or Pull Request.

> Maybe in Network Configuration Management?

Seems appropriate.

Reply:
@nodiscc: created #449